src/chi/components/alert/_banner.scss

```diff
@@ -6,63 +6,21 @@
       border-radius: 0;
       border-width: 0.0625rem 0;
       padding-left: 1rem;
-      padding-right: 2rem;
+      padding-right: 1rem;
 
-      &.-small {
-        display: flex;
-        justify-content: center;
-        padding-left: 2rem;
-
-        &::before {
-          display: none;
-        }
+      &.-dismiss {
+        justify-content: space-between;
       }
 
       &.-large {
         padding-bottom: 1.1875rem;
         padding-top: 0.75rem;
-
-        &::before {
-          font-size: $text-large;
-          line-height: $line-height;
-          margin-bottom: 0.4375rem;
-        }
       }
 
-      @each $state in map-keys($states) {
-        &.-#{$state} {
-          &.-hasIcon {
-            flex-direction: row-reverse;
-            padding-left: 3.5rem;
-
-            &::after {
-              left: 1.125rem;
-            }
-
-            &.-small {
-              padding-left: 2rem;
-
-              &::after {
-                height: 1rem;
-                left: initial;
-                margin-right: 0.5rem;
-                position: relative;
-                top: initial;
-                transform: initial;
-                width: 1rem;
-              }
-            }
-
-            &.-large {
-              &::after {
-                height: 1.5rem;
-                left: 1rem;
-                top: 0.9375rem;
-                transform: initial;
-                width: 1.5rem;
-              }
-            }
-          }
+      & > .-close {
+        &.-small {
+          line-height: .75rem;
```

Comment:
This `.75rem` line-height should be one level further down on `a-icon`, not its parent.

Reply:
fixed

Reply:
this doesn't appear to have been fixed - line-height is still rendering as 1rem

![image](https://user-images.githubusercontent.com/130654/49244777-73b2ba80-f3c5-11e8-8209-6b48dbc1f2e5.png)

Reply:
you're right. fixed now. sorry